airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ResourceNotAvailable/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { faArrowDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FormattedMessage } from "react-intl";
+
+import { Button } from "components/ui/Button";
+import { Text } from "components/ui/Text";
+
+import { SourceDefinitionRead } from "core/request/AirbyteClient";
+import { useDocumentationPanelContext } from "views/Connector/ConnectorDocumentationLayout/DocumentationPanelContext";
+
+import styles from "./ResourceNotAvailable.module.scss";
+import octaviaWorker from "../octavia-worker.png";
```

Comment:
@Upmitt Could we make sure to get a proper illustration here (vector art!). We don't want to add new `png` images into the app, so would appreciate if we can exchange that via a proper illustration.